packages/Multiselect/stories/Multiselect.stories.tsx

```diff
@@ -1,35 +1,62 @@
-import React from "react";
 import { storiesOf } from "@storybook/react";
-import Multiselect from "../src";
+import React, { useEffect, useState } from "react";
 import MultiSelectReadme from "../README.md";
-const data = {
-  keyValue: "name",
-  filterBy: ["name", "description"],
-  data: [
-    {
-      id: 1,
-      name: "Blaze",
-      description: "Lorem ipsum dolor."
-    },
-    {
-      id: 2,
-      name: "KP",
-      description: "Aliquam tincidunt."
-    },
-    {
-      id: 3,
-      name: "Pulser",
-      description: "Vestibulum auctor."
-    }
-  ]
-};
-storiesOf("Multiselect", module)
-  .addParameters({
-    readme: {
-      sidebar: MultiSelectReadme
+import Multiselect from "../src";
+
+const DemoComponent = () => {
+  const [multiselectData, setMultiselectData] = useState({
+    keyValue: "name",
+    filterBy: ["name", "description"],
+    data: [
+      {
+        id: 1,
+        name: "Blaze",
+        description: "Lorem ipsum dolor."
+      }
+    ]
+  });
+  const [a, setA] = useState(false);
+
+  const registerSetTimeOut = () => {
+    setTimeout(() => {
+      console.log("TIMEOUT");
```

Comment:
Please remove console.log and then feel free to merge

Reply:
Sure! 👍